src/getHumanAge.test.js

```diff
@@ -3,8 +3,21 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should convert cat and dog years to human years correctly', () => {
```

Comment:
What does 'correctly' means? Please, add more details, provide the numbers. Also few postive test should be written with that data